pkgs/development/python-modules/rdflib-jsonld/default.nix

```diff
@@ -0,0 +1,23 @@
+{ buildPythonPackage
+, fetchurl
+, lib
+
+, rdflib
+}:
+buildPythonPackage rec {
+  name = "rdflib-jsonld";
+  version = "0.5.0";
+  src = fetchurl {
+    url = "https://files.pythonhosted.org/packages/f5/17/45e137be0d93b70827fe5529c0400731344a978bc792193d7d9152e6dbe4/${name}-${version}.tar.gz";
+    sha256 = "4f7d55326405071c7bce9acf5484643bcb984eadb84a6503053367da207105ed";
+};
+  doCheck = true;
+  format = "setuptools";
+  propagatedBuildInputs = [ rdflib ];
```

Comment:
```suggestion
  nativeBuildInputs = [ nose ];

  propagatedBuildInputs = [ rdflib ];
```

```
  error: Command '['/nix/store/fjgnz0xfl04hsblsi4ym5y5akfh6mlmy-python3-3.8.5/bin/python3.8', '-m', 'pip', '--disable-pip-version-check', 'wheel', '--no-deps', '-w', '/build/tmpf7v5k5qr', '--quiet', 'nose']
```

Reply:
What command told you that `nose` was missing in my dependencies ?

Reply:
`'--no-deps', '-w', '/build/tmpf7v5k5qr', '--quiet', 'nose']` it's at the end